sys-utils/setpriv-landlock.h

```diff
@@ -0,0 +1,41 @@
+#ifndef UTIL_LINUX_SETPRIV_LANDLOCK
```

Comment:
Again, copyright and license.

Reply:
See for example, disk-utils/addpart.c (9e95aa1298d67331cab00f385d7e0cb9c64a263b).